Translation corrections (#148)
07th-mod/higurashi-console-arcs: Update/some_014.txt

//rそれでも、…重く全身にのしかかる無力感に髪をかきむしって苛立ち、暁はぎりっ、と奥歯をかみ締める。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "それでも、…重く全身にのしかかる無力感に髪をかきむしって苛立ち、暁はぎりっ、と奥歯をかみ締める。",
			NULL, "Even so... Akira clenched his jaw while scratching his hair to relieve the irritating helpless feeling encroaching on his entire body.", Line_Normal);
	ClearMessage();

//r…本当に、彼は無力だった。気まぐれに人の運命を弄ぶ神に対してよりも、…何もできなかった自分に、怒りがこみ上げておさまらない。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…本当に、彼は無力だった。気まぐれに人の運命を弄ぶ神に対してよりも、…何もできなかった自分に、怒りがこみ上げておさまらない。",

> 			NULL, "...He truly was helpless. Anger ignited and raged like a wildfire in his chest- directed less at the god that meddled with humans' fate and more at how powerless he had been.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...He truly was helpless. Anger ignited and raged like a wildfire in his chest - directed less at the god that meddled with humans' fate and more at how powerless he had been.", Line_Normal);
```

//rその憎々しいほど歪みきった表情に、思わず暁の握り締めたこぶしが小刻みに震えたが、…それは事実で、間違いなく図星だった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "その憎々しいほど歪みきった表情に、思わず暁の握り締めたこぶしが小刻みに震えたが、…それは事実で、間違いなく図星だった。",

> 			NULL, "Akira clenched his fists instinctively and trembled as his face warped into a loathsome expression... but he couldn't deny any of it. Everything his doppleganger was right on the mark.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Akira clenched his fists instinctively and trembled as his face warped into a loathsome expression... but he couldn't deny any of it. Everything his doppelganger said was right on the mark.", Line_Normal);
```

//r夏美はふらつきながらも頑として、暁を自宅に近づけようとしなかった。…いや、見せようともしなかった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "夏美はふらつきながらも頑として、暁を自宅に近づけようとしなかった。…いや、見せようともしなかった。",
			NULL, "Despite her stupor, Natsumi had stubbornly tried to keep Akira away from her home. ...No, she didn't want him to see it.", Line_Normal);
	ClearMessage();

//rそして暁は、…彼女の噂を思い出して深入りを避け、躊躇いながらもそこで別れてしまった……。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そして暁は、…彼女の噂を思い出して深入りを避け、躊躇いながらもそこで別れてしまった……。",

> 			NULL, "And Akira... remembering the rumors about her and unwilling to get deeply involved, reluctantly parted with her there......", Line_Normal);

Reviewer: ```suggestion
			NULL, "And Akira... remembering the rumors about her and unwilling to get deeply involved, reluctantly parted ways with her there......", Line_Normal);
```
